broker-plugin/broker-cli/pom.xml

```diff
@@ -0,0 +1,72 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <parent>
+    <groupId>io.enmasse</groupId>
+    <artifactId>broker-plugin</artifactId>
+    <version>0.30-SNAPSHOT</version>
+  </parent>
+  <modelVersion>4.0.0</modelVersion>
+  <artifactId>broker-cli</artifactId>
```

Comment:
Are we intending to publish this Jar (maven-central)?

Reply:
To keep things simple, I think we should only use it as part of the broker image for now.

Reply:
That's fine.. I just wanted to understand our intended direction.  I think the justification for publication would need to be strong, as it brings with it a lot of work.